### DISCUSSION TITLE
How to hide this command-line window？

### DISCUSSION THREAD

**Discussion by zhanjunjie2019** [fyne-io/fyne#5208] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/f88973f7-2f36-42f2-b9f4-db72ff61b0ed)
How to hide this command-line window？

**Comment by Jacalz** [comment, marked as answer]
That is a side effect of how Windows applications are built. If you build your app with `go build -ldflsgs="-H windowsgui"`, or use the `fyne build` or `fyne package` commands, you get rid of it.
